packages/vkui/src/components/Checkbox/CheckboxInput/CheckboxInput.tsx

```diff
@@ -16,6 +16,12 @@ import { RootComponent } from '../../RootComponent/RootComponent';
 import { VisuallyHidden } from '../../VisuallyHidden/VisuallyHidden';
 import styles from './CheckboxInput.module.css';
 
+type VendorIconOnCompactType = typeof Icon20CheckBoxOn;
+type VendorIconOnRegularType = typeof Icon24CheckBoxOn;
+type VendorIconOffCompactType = typeof Icon20CheckBoxOff;
+type VendorIconOffRegularType = typeof Icon24CheckBoxOff;
+type VendorIconIndetermanateType = typeof Icon20CheckBoxIndetermanate;
```

Comment:
```suggestion
type VendorIconType = typeof Icon20CheckBoxOn;

export type CheckboxInputIconType = React.ComponentType<React.SVGProps<SVGSVGElement>> | VendorIconType;
```

у всех иконок тип один, поэтому не нужно под каждый

давай ещё вынесем в общий экспоритуремый тип `CheckboxInputIconType` и будем ниже переиспользовать, который нужно будет в `packages/vkui/src/index.ts` добавить в ре-экспорт

Reply:
Поправил